### PR TITLE
jhiccup: fix broken url

### PR DESCRIPTION
**Pull request by vszakats**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/jhiccup.rb [hunk 1]
```diff
@@ -1,7 +1,7 @@
 class Jhiccup < Formula
   desc "Measure pauses and stalls of an app's Java runtime platform"
-  homepage "http://www.jhiccup.com"
-  url "http://www.azulsystems.com/sites/default/files/images/jHiccup.1.3.7.zip"
+  homepage "http://www.jhiccup.com/"
```
Comment (fxcoudert) [inline Formula/jhiccup.rb]: This is now a redirect to `https://www.azul.com/products/open-source-tools/jhiccup-performance-measurement-tool/`

**Review by fxcoudert — CHANGES_REQUESTED** [review]

File: Formula/jhiccup.rb (hunk 1 above)
Comment (vszakats) [inline Formula/jhiccup.rb]: Noticed, but opted to leave it as-is. Will update now.

Comment (vszakats) [inline Formula/jhiccup.rb]: I've committed (+ squashed) the requested change.

**Comment by fxcoudert** [comment]
Thanks @vszakats for your contribution!

**Pull request merged**